src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -865,7 +865,7 @@ def check_connectivity(url='https://azure.microsoft.com', max_retries=5, timeout
 
 def send_raw_request(cli_ctx, method, url, headers=None, uri_parameters=None,  # pylint: disable=too-many-locals,too-many-branches,too-many-statements
                      body=None, skip_authorization_header=False, resource=None, output_file=None,
-                     generated_client_request_id_name='x-ms-client-request-id'):
+                     generated_client_request_id_name='x-ms-client-request-id', disable_request_response_logging=False):
```

Comment:
check if we actually need this flag or how we're doing without it in other apis.

Reply:
For use in other places, it is logging out the entire request & response header + content(for eg. if we do the deployment using `src-url`, it uses the ARM endpoint and uses this method to deploy. I think logging the headers and body will be too verbose for this use case, since we will be polling the deploymentstatus ARM endpoint every few seconds.